cns/cni-telemetry-sidecar/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/Azure/azure-container-networking/cns/logger"
+)
+
+var (
+	version    = "unknown"
+	configPath = flag.String("config", "/etc/cns/cns-config.json", "Path to CNS configuration file")
+)
+
+func main() {
+	flag.Parse()
+
+	// Initialize logging for the CNI telemetry sidecar
+	logger.InitLogger("azure-cns-cni-telemetry-sidecar", 1, 1, "/var/log/azure-cns-telemetry")
```

Comment:
should the naming be the same-- would it be cns or cni telemetry? Like below the logger prints that it's azure cni telemetry sidecar but the log is to the file azure-cns-telemetry